src/sanitizers.rs

```diff
@@ -128,9 +130,52 @@ pub fn strip_stuff_between_messages(s: &str) -> String {
     // Do a third pass to actually copy only the wanted lines from the input to the output.
     s.lines()
         .enumerate()
-        .filter(|(i, _)| !lines_to_delete.contains(i))
-        .map(|(_, line)| format!("{}\r\n", line))
-        .collect()
+        .filter_map(|(i, contents)| (!lines_to_delete.contains(&i)).then(|| contents))
```

Comment:
Did you benchmark this vs your previous solution?

Reply:
Yes, it is slightly faster, when running all sanitizers I get about 8.0ms vs 8.8ms with the previous solution.